gradle.properties

```diff
@@ -1,2 +1,2 @@
 group=com.heapwhisperer.metrics.jvm
-version=1.0.2
+version=1.0.2-rocana2.0.0-SNAPSHOT
```

Comment:
Bashar [said](https://github.com/gburton1/metrics-jvm-nonaccumulating/pull/1#discussion_r85027764)
> Does this mean we have to make a release build of this dependency before we release 2.0.0?

Joey replied:
> Yup, that's the pattern we take with java library forks like Kite.

I really like the idea of being consistent in how we do things.

On the other hand, it's not bad to reflect on how we do things and make sure it's the most efficient.

Recently we've revisited this process with the Impala fork and are having second thoughts.